specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/VirtualNetworkRulesDelete.json

```diff
@@ -10,11 +10,7 @@
     "200": {
       "body": ""
```

Comment:
Model validation is also complaining about 200 (comment applies to all three examples).

Reply:
This has been resolved